### PR TITLE
Continued refinement

### PR DESCRIPTION
**Pull request by jakeatdocforce**
Homepage icon padding
footer hover
footer border
padding adjustments to re center content

### CODE REVIEW

File: tool/xrpl.scss [hunk 1]
```diff
@@ -275,13 +281,16 @@ h3 a:hover {
 }
 
 .xrpl-footer {
-  border-top: 1px solid $black;
```
Comment (mDuo13) [inline tool/xrpl.scss]: On content pages, this leaves no border above the footer. I think you might want to remove the bottom border of `.landing section:last-of-type` instead.

File: assets/css/devportal.css [hunk 2]
```diff
@@ -156,10 +156,14 @@ body {
   -webkit-font-smoothing: antialiased;
   font-size: 0.9375rem;
 }
+.row {
+  margin: 0 -1px;
```
Comment (mDuo13) [inline assets/css/devportal.css]: Just curious, but what's this compensating for? Is it the 2px of padding on line 299?

File: tool/xrpl.scss [hunk 3]
```diff
@@ -197,22 +197,28 @@ h1, h2, h3, h4, h5 {
 /* Some styles that use the colors defined above ---------------------------- */
 h1 a,
 h2 a,
-h3 a {
+h3 a,
+.xrpl-footer h5 a {
```
Comment (mDuo13) [inline tool/xrpl.scss]: I think adding `xrpl-footer h5 a` to this declaration caused the footer headers to lose their text decoration on hover, so you might want to add another rule to get that back.

Comment (jakeatdocforce) [reply tool/xrpl.scss]: Moved that to a separate rule.

File: assets/css/devportal.css (hunk 2 above)
Comment (jakeatdocforce) [reply assets/css/devportal.css]: I believe so. It was part of the changes Ohn posted.  Without it doesn't quite match up with the top nav bar

File: tool/xrpl.scss (hunk 1 above)
Comment (jakeatdocforce) [reply tool/xrpl.scss]: you are correct. I changed it. Footer border restored on content pages.

File: assets/vendor/bootstrap.css [hunk 4]
```diff
@@ -5459,7 +5459,7 @@ button.bg-dark:focus {
   z-index: 1;
   pointer-events: auto;
   content: "";
-  background-color: rgba(0, 0, 0, 0); }
+  background-color: transparent; }
```
Comment (mDuo13) [inline assets/vendor/bootstrap.css]: It's funny, whatever sass compiler you're using likes writing `transparent`, whereas the sass compiler I'm using keeps wanting to switch it to `rgba(0, 0, 0, 0)` instead. ¯\_(ツ)_/¯

**Pull request merged**